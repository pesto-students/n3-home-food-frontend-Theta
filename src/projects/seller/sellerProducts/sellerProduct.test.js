import * as React from "react";
import Enzyme, { shallow } from "enzyme";
import EnzymeAdapter from "enzyme-adapter-react-16";
import SellerProducts from "./sellerProducts";

Enzyme.configure({ adapter: new EnzymeAdapter() });

describe("Seller products Dashboard component", () => {
  let wrapper;
  test("render without error", () => {
    wrapper = shallow(<SellerProducts   />);
    expect(wrapper.length).toBe(1);
  });

  test('should render correctly in "debug" mode', () => {
    const component = shallow(<SellerProducts  />);

    expect(component).toMatchSnapshot();
  });
});