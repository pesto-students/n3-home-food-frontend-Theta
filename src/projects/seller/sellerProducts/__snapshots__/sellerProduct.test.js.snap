// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seller products Dashboard component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
